src/Chapter19/Cryptographer.cs

```diff
@@ -111,7 +111,7 @@ public async Task DecryptAsync(byte[] encryptedData, Stream outputStream)
             await DecryptAsync(encryptedData, CryptoAlgorithm.CreateDecryptor(), outputStream);
         }
 
-        public static async Task<string> DecryptAsync(byte[] encryptedData, byte[] key, byte[] iV)
+        static public async Task<string> DecryptAsync(byte[] encryptedData, byte[] key, byte[] iV)
```

Comment:
Is there a reason for putting static before public here? It is [more common to have the access modifier first](https://stackoverflow.com/questions/191929/is-there-a-convention-to-the-order-of-modifiers-in-c)

Reply:
Hmmm.... Great question... and something we should include in IntelliTect analysis.
I prefer static first because it stands from the normal (which is non-static). In other words, by putting static first, it is more noticeable than when it is buried in front of public.  That said, I have seen code analysis rules suggesting to put public first - making this another issue similar to private field naming.

Reply:
I would say log something in the [Coding Standards](https://github.com/IntelliTect/CodingStandards) and label it with "proposal". I think before we change from what the default editorconfig and other static tools suggest we should have a good reason though.

Reply:
Done: https://github.com/IntelliTect/CodingStandards/issues/42